### PR TITLE
Added cli-changes second itr

### PR DESCRIPTION
**Pull request by MG190202**

### CODE REVIEW

File: dozer-ingestion/oracle/src/connector/mod.rs [hunk 1]
```diff
@@ -524,7 +524,7 @@ mod tests {
         let data_user = "DOZER";
         let host = "database-1.cxtwfj9nkwtu.ap-southeast-1.rds.amazonaws.com";
         let sid = "ORCL";
-        let pid = None;
+        let pid = sid;
```
Comment (Jesse-Bakker) [inline dozer-ingestion/oracle/src/connector/mod.rs:527]: This seems unrelated

Comment (MG190202) [reply dozer-ingestion/oracle/src/connector/mod.rs]: Yes,but the lint error shows it, i have undone this in the check2 commit though

Comment (MG190202) [reply dozer-ingestion/oracle/src/connector/mod.rs]: here the linter shows the same error ,should i ignore it?
https://github.com/getdozer/dozer/actions/runs/7812319344/job/21309138408?pr=2379

**Comment by Jesse-Bakker** [comment]
Rebased and merged in #2382
